cmd/zc_enumerator.go

```diff
@@ -65,14 +67,15 @@ func (storedObject *storedObject) isMoreRecentThan(storedObject2 storedObject) b
 }
 
 // a constructor is used so that in case the storedObject has to change, the callers would get a compilation error
-func newStoredObject(name string, relativePath string, lmt time.Time, size int64, md5 []byte, blobType azblob.BlobType) storedObject {
+func newStoredObject(name string, relativePath string, lmt time.Time, size int64, md5 []byte, blobType azblob.BlobType, containerName string) storedObject {
```

Comment:
I notice that this parameter is always "" in all calls. It's only the middleman processor/decorator that ever sets the value.  So what about removing the param here, and all the "// not needed" comments at the call sites?  Just let this field be something that's only set, in all the cases where its needed, by the middleman/decorator.

Reply:
Fair point.